jscomp/test/class4_test.js

```diff
@@ -95,7 +95,9 @@ function restricted_point2$prime_init($$class) {
 
 var restricted_point2$prime = CamlinternalOO.make_class(shared$2, restricted_point2$prime_init);
 
-var Point = /* module */[/* restricted_point' */restricted_point];
+var Point = /* module */{
+  "restricted_point'": restricted_point
+};
```

Comment:
For the record, is escaping here worth  or not `{ restricted_point$prime : restricted_point}`

Reply:
@bobzhang what the situation at the moment if one declares an explicit object type with a field name containing a primed name? I think we should try to get a unified experience.